src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -35,7 +40,60 @@ public WorkflowJob(Injector injector, URL url, String name) {
         super(injector, url, name);
     }
 
-    public final Control script = control("/definition/script");
+    public final Control script = new Control(this, "/definition/script") {
+        @Override
+        public void set(String text) {
+            try {
+                super.set(text);
```

Comment:
So is this even needed now, considering it is always using the latest version of the plugin, which will have the ACE Editor?